src/css/object/project/job-listings.scss

```
@@ -0,0 +1,68 @@
+@use '../../object/utility/utility.scss';
+
+.p-joblistings {
+  margin-top: 60px;
+  width: 1600px;
+  height: 637px;
+  background-color:#DEEEF9 ;
+}
+
+.new-listings {
+  font-size: 25px;
+  padding-top: 67px;
+  padding-left: 210px;
+}
```

以下のような記述にして、すべての class に `p-` がつくようにお願いします。
すべて完成した後で、デザインの変更などをするとなった時に、HTMLからSassのスタイルの定義場所を簡単に特定させるためです。

p-joblistings__new-listings というクラスになっていると、project 配下の joblistings.scss の new-listings のクラスについてそうだということが特定できますよね。

全ファイルにわたってまずはこちらの修正をお願いします

```suggestion
.p-joblistings {
  margin-top: 60px;
  width: 1600px;
  height: 637px;
  background-color:#DEEEF9 ;
  
  &__new-listings {
    font-size: 25px;
    padding-top: 67px;
    padding-left: 210px;
  }
}
```

---

修正致しました。
d18392b897533004283418b749e82090615e765c

---

木原さんの「全ファイルに渡って」という内容がコミット内容見てる限り反映されてなさそうなので、見直しお願いします！

---

修正しました。
9f837c1137377800bf761181d760ef44577978b9